src/Components/Api.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios"
+
+const api = axios.create({
+    baseURL: "https://sabrinas-nc-news.onrender.com/api/",
+})
+
+const getTopics = () => {
+    return api.get("/topics").then(({ data }) => {
+        return data.topics
+      });
+}
```

Comment:
Very nice to see _Axios_ being used and especially with the syntax for a `baseURL` 👏